src/eventshub/src/component/Autocomplete/Autocomplete.tsx

```diff
@@ -1,132 +1,134 @@
-import { createElement, Fragment, useEffect, useRef, useState } from 'react';
+import { createElement, Fragment, useEffect, useRef } from 'react';
 import {
   autocomplete,
   AutocompleteOptions,
   getAlgoliaResults,
 } from '@algolia/autocomplete-js';
-import { BaseItem } from '@algolia/autocomplete-core';
 import type { SearchClient } from 'algoliasearch/lite';
 import { usePagination, useSearchBox } from 'react-instantsearch-core';
+import { nanoid } from 'nanoid';
+import { useAppState } from '../../hooks';
+import { algoliaSuggestionsIndexMap } from '../../redux/slices/appSlice';
+import type { AutocompleteComponents } from '@algolia/autocomplete-js';
 import type { Root } from 'react-dom/client';
 import { createRoot } from 'react-dom/client';
+import { AlgoliaVenueItem } from '../../utilities/types';
 
 // TODO: Determine if we should build upon or remove algolia styles
-// import '@algolia/autocomplete-theme-classic';
+import '@algolia/autocomplete-theme-classic';
```

Comment:
Added this theme so the drop down shows up near the search bar. You can remove this if you want and style the drop down. You can also move where the drop down box gets rendered using the panelContainer param.